frontend/src/libs/packages/map/map.package.ts

```diff
@@ -36,6 +37,8 @@ class MapService implements IMapService {
 
   private routes!: google.maps.DistanceMatrixService;
 
+  private autocomplete!: google.maps.places.AutocompleteService;
```

Comment:
Also here, we may need to use something else from the library, and here it's just Autocomplete.
```suggestion
  private places!: google.maps.places.placesLibrary;
```